rpc/rpc-wrappers.go

```diff
@@ -878,6 +880,26 @@ func NewStorageAuthorityServer(rpc Server, impl core.StorageAuthority) error {
 		return
 	})
 
+	rpc.Handle(MethodGetAuthorizationsByDomain, func(req []byte) (response []byte, err error) {
+		ident := core.AcmeIdentifier{}
+		err = json.Unmarshal(req, &ident)
+		if err != nil {
+			return nil, err
+		}
+		authz, err := impl.GetAuthorizationsByDomain(ident)
+		if err != nil {
+			return
```

Comment:
This function should be consistent about naked returns vs clothed returns. I'd suggest the latter (and remove the names from the func declaration)